velox/type/Conversions.h

```diff
@@ -57,6 +57,52 @@ struct Converter {
   }
 };
 
+/// This is based on Presto java's castToBoolean method.
```

Comment:
I assume this is not public API, hence, should use //. Can this be a private method of a Converter struct?